test/e2e/windows/hybrid_network.go

```diff
@@ -79,6 +72,28 @@ var _ = SIGDescribe("Hybrid cluster network", func() {
 
 		})
 
+		ginkgo.It("should provide Internet connection for containers using DNS [Feature:Networking-DNS]", func() {
+			linuxPod := createTestPod(f, linuxBusyBoxImage, linuxOS)
+			ginkgo.By("creating a linux pod and waiting for it to be running")
+			linuxPod = f.PodClient().CreateSync(linuxPod)
+
+			ginkgo.By("verifying pod external connectivity to the internet")
+
+			ginkgo.By("checking connectivity to 8.8.8.8 53 (google.com) from Linux")
+			assertConsistentConnectivity(f, linuxPod.ObjectMeta.Name, linuxOS, linuxCheck("8.8.8.8", 53))
```

Comment:
Hmm, I'm not sure why we had ``assertConsistentConnectivity`` from the Linux container to ``8.8.8.8`` before, but I think it would make more sense to have it from the Windows container to ``8.8.8.8`` instead, since this is ``test/e2e/windows`` after all.

Reply:
@claudiubelu as it is mentioned [here](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/windows/hybrid_network.go#L52) that here we are checking internet connectivity for both Windows and Linux pods that's why have` assertConsistentConnectivity from the Linux container to 8.8.8.8`. Please correct me if I am wrong here. And also thanks for your input.

Reply:
I was confused at the beginning too, but seems this whole test block is for hybrid, not windows only

Reply:
> @claudiubelu as it is mentioned [here](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/windows/hybrid_network.go#L52) that here we are checking internet connectivity for both Windows and Linux pods that's why have `assertConsistentConnectivity from the Linux container to 8.8.8.8`. Please correct me if I am wrong here. And also thanks for your input.

Right, but in that case, it was testing Linux -> Internet (8.8.8.8), Windows -> Internet (google.com), and after that Linux -> Windows, and then Windows -> Linux. After all, I think it made sense in that scenario; it was a good idea to make sure both Linux and Windows network connections were properly set up before testing the Linux - Windows connection. If you'd test the Linux - Windows connection directly and it'd fail, you wouldn't know who's fault it is.

Looking back in history, the most important part is testing out the Linux -> Windows and Windows -> Linux connectivity (https://github.com/kubernetes/kubernetes/issues/74377), the other checks were made to ensure networking works in both pods in the first place. I don't think that connecting to 8.8.8.8 or google.com is relevant for the test, IMO. I think connecting to the Kubernetes API instead should be sufficient. @jayunit100 , what do you think?

Reply:
> , it was testing Linux -> Internet (8.8.8.8), Windows -> Internet (google.com), and after that Linux -> Windows, and then Windows -> Linux. After all, I think it made sense in that scenario; it was a good idea to make sure both Linux and Windows network connections were properly set up before testing the Linux - Windows connection.

> the other checks were made to ensure networking works in both pods in the first place

internet connectivity from pods doesn't guarantee that intra-pod communication works, internet connectivity should be a separate test case that can be filtered,  this is why the bug was opened, doesn't it?

Reply:
right, we want to test both of these things. In some contexts it makes sense not to expect internet connectivity, but it is a useful check to see that we didn't royally break something in say, kube-proxy :-)

putting external network connectivity in a separate tagged test will allow excluding them in environments where the test failing is working as intended